### DISCUSSION TITLE
yarn build / generate: style not applied if not in template files?

### DISCUSSION THREAD

**Discussion by josephlevarato** [nuxt/nuxt#16589] [Questions] [answered]
Hello,

I'm currently trying to deploy a website made with Nuxt 3 in server mode.

I've had issues with environment variables so I tried on my own computer to try and troubleshoot any issue, however when I run `yarn build` or `npx nuxi build`, I realized that I had no style, except for a few components where the style was in the `<style>` part of the template file.

Most of the style I have is dispatched in a few scss files, and called as follows in the `nuxt.config.ts` file:

```
css: ["@/assets/styles/main.scss"]
```

 In a `yarn dev` context, everything works perfectly.

I can see some style in my `<head>` that I can recognize, but nothing is applied and I can't tell for sure if everything is there.

How can I fix that?

Thank you in advance

**Comment by TheAlexLichter** [comment]
Can you provide a small reproduction?

**Comment by josephlevarato** [comment, marked as answer]
I can't reproduce my issue on another computer, it might be because of that, or because the RC 12 has been released. In any case, it's no longer a problem!
